UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/TerrainMenu.cs

```diff
@@ -5,10 +5,13 @@ public class TerrainMenu : IMenu
 {
     private Vector2Int terrainSize;
     private TerrainGenerator terrainGenerator;
+    private OverlayGenerator gridGenerator;
 
-    public TerrainMenu() {
+    public TerrainMenu()
+    {
         terrainSize = new Vector2Int(10, 10);
         terrainGenerator = new TerrainGenerator();
+        gridGenerator = new OverlayGenerator();

```

Comment:
consistency?

Reply:
in this case we are in the terrain menu where I am using the overlaygenerator only as a grid generator (to align it to the terrain). This name provides more information/context but I can see the confusion. What do you think?

Reply:
dunno - there's also the CreateGrid() function that creates OverlayGenerator.GenerateGridForTerrain(DEFAULT_TERRAIN); etc

Reply:
cool, renamed it